src/js/wrap.cpp

```diff
@@ -19,1580 +18,1603 @@
 #include <quickjs/quickjs-exports.h>
 #include <quickjs/quickjs.h>
 
-namespace js
+namespace ccf
```

Comment:
You can do `namespace ccf::js` to remove the indent diff here/elsewhere.

Reply:
Thanks, did not know that!